### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Jessekay**
- I designed desktop version using media queries(Desktop: >=768px) in my CSS file,
- I used grid and flexbox to lay out elements on the page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,141 +42,167 @@ <h1 class="title">I’m Jesse<br> Glad to see<br> you!</h1><br><br>
...
         </div>
-        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <p class="text2">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
         <ul class="skills">
           <li>html</li>
+          <li>Ruby on rails</li>
           <li>css</li>
           <li>Javascript</li>
         </ul>
         <div class="but"><button class="button">See project</button></div>
```
Comment (mckent05) [inline index.html:86]: - You have done well making sure your site is responsive across desktop screens, but in order to make sure your site looks closely like the design template, I suggest that you implement the button interaction styling i.e :hover, :active, e.tc for all your buttons, just like you did for the form button, please fix this 🙏 .
Button Interaction styling:
interactions

- For your mobile version, you seem to have introduced a bug in your design. Please notice the marked section in the screenshot below and ensure they look like the design template
Your project:
gnfgndgn

File: index.html [hunk 2]
```diff
@@ -31,141 +42,167 @@ <h1 class="title">I’m Jesse<br> Glad to see<br> you!</h1><br><br>
...
+          <input type="email" class="input-email" name="email_address" id="email_address"
+            placeholder="Enter your email address" required>
+        </label>
+        <label for="text_message">
+          <textarea name="message" id="text-message" cols="30" rows="10" placeholder="Write your message here"
+            maxlength="500" required></textarea>
+        </label>
+        <button id="contact-btn" class="btn contact-btn" type="submit"> Get in touch</button>
+        </form>
+      </footer>
```
Comment (mckent05) [inline index.html:205]: - The mobile version of the contact form does not look like the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project:
ndgnzdn

File: index.html [hunk 3]
```diff
@@ -31,141 +42,167 @@ <h1 class="title">I’m Jesse<br> Glad to see<br> you!</h1><br><br>
...
-            <img class="lang-img" src="./images/html.png" alt="html"> 
+            <img class="lang-img" src="./images/html.png" alt="html">
             <span class="span">HTML</span>
           </li>
           <li class="sl">
-            <img class="lang-img" src="./images/css.png" alt="css"> 
+            <img class="lang-img" src="./images/css.png" alt="css">
             <span class="span">CSS</span>
           </li>
         </ul>
```
Comment (mckent05) [inline index.html:174]: - The mobile version of the `about-myself` section seems to have a bug, please fix this 👍. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project:
mxfmx

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @Jessekay, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
- Please, I suggest you make your site responsive across all mobile screens `<768px`. This would ensure that your screen fits properly across all mobile screen sizes. Please fix this 😄 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -1,28 +1,39 @@
...
+          <li>Contact</li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <section class="wrapper">
+    <h1 class="title">I'm Jesse Glad to see you!</h1>
+    <p class="intro">I'm a software developer! I can help you build a product , feature or website Look through some of
+      my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+    </p><br>
```
Comment (Alfredbis29) [inline index.html:34]: - [ OPTIONAL] Nice Design but there is still something to address, I noticed that your heading and paragraph should be on the same line as we can see that on Figma design, kindly fix this for your project to look more attractive :wink: 
Your Design 
Screenshot from 2023-02-25 19-29-33

Figma Design 
Screenshot from 2023-02-25 19-28-21

File: index.html [hunk 5]
```diff
@@ -31,141 +42,167 @@ <h1 class="title">I’m Jesse<br> Glad to see<br> you!</h1><br><br>
...
-        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <p class="text1">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
         <ul class="skills">
           <li>html</li>
+          <li>Ruby on rails</li>
           <li>css</li>
           <li>Javascript</li>
         </ul>
         <div class="but"><button class="button">See project</button></div>
       </div>
```
Comment (Alfredbis29) [inline index.html:87]: - [ ] As required in the previous review [here](https://github.com/Jessekay/My-portfolio/pull/4/files#r1117933857), your website should be responsive, kindly make your website responsive to adapt to the user's device, and it makes it easier to use and navigate. Remember that this leads to a better user experience, which can increase the time users spend on your site, reduce bounce rates, and increase conversion rates. :wink: 
Screenshot from 2023-02-25 19-38-48

File: index.html [hunk 6]
```diff
@@ -1,28 +1,39 @@
...
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <section class="wrapper">
+    <h1 class="title">I'm Jesse Glad to see you!</h1>
+    <p class="intro">I'm a software developer! I can help you build a product , feature or website Look through some of
+      my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+    </p><br>
+    <a href="#" class="connect">LET'S CONNECT</a><br>
```
Comment (Alfredbis29) [inline index.html:35]: - [ OPTIONAL]KIndly try to match your design to the figma one , I can see that your 'LET'S CONNECT' button is not looking the same as the one the Figma design when we move on mobile version  :wink: 
Your Design
Screenshot from 2023-02-25 19-46-19

Figma Design
Screenshot from 2023-02-25 19-47-06

_Note: I can see that even your `h2` (About Myself) also is not the same as on Figma design_

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
### Hi @Jessekay   ,
 You did a great job so far 👏🏼 👏🏼 There are some issues that you still need to work on to go to the next project but you are almost there!
 
 
 ### To highlight 💪🏻
  *Good Commit messages :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 * Good Job so far:+1:  

## STATUS: CHANGES REQUESTED 
 
_Please Check the comments under the review and make all required changes_
 
## Optional suggestions

- [ ] Kindly follow the Project Guidelines and  I'm sure that you will make it , then when you are ready you can request for another review :wink:  :muscle: 


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 Cheers and Happy coding! :clap: :clap: :clap: :computer: Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please ping me @alfredbis29 when you comment so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Jessekay, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**